.node-version

```diff
@@ -1 +1 @@
-v10.2.0
+v10.15.3
```

Comment:
v10.2.0 is an integrated Node with VS Code, but it raises error of `fsevents` when running `yarn watch`. So we updated Node version to the latest LTS.

A test environment in CI that is not required build keeps using v10.2.0.